core/domain/summary_services.py

```diff
@@ -339,6 +451,22 @@ def _get_displayable_collection_summary_dicts(collection_summaries):
 def get_library_groups(language_codes):
     """Returns a list of groups for the library index page. Each group has a
     header and a list of dicts representing activity summaries.
+
+    Args:
+        language_codes: list(str). A list of language code
```

Comment:
code --> codes.

Also, what is the relevance of this arg? Explain it a bit -- how does it affect the result?